code/datums/spells/construct_spells.dm

```diff
@@ -65,11 +65,27 @@
 	action_background_icon_state = "bg_demon"
 
 	summon_type = list(/obj/item/device/soulstone)
-
+	

```

Comment:
not sure if this is the cause of any issues, but there is a random tab on an otherwise blank line here